pkg/provisioner/ironic/ironic.go

```diff
@@ -609,6 +609,59 @@ func (p *ironicProvisioner) UpdateHardwareState() (result provisioner.Result, er
 	return result, nil
 }
 
+func (p *ironicProvisioner) startManualCleaning(ironicNode *nodes.Node) (result provisioner.Result, err error) {
+	var cleanSteps []nodes.CleanStep
+
+	if p.bmcAccess.RAIDInterface() != "" {
```

Comment:
I am sorry, i really want to have functions return quickly, but i think we couldn't do it. Because, we also need do something like `BIOS` in `startManualCleaning`. Even if `RAID` is not supported, `BIOS` should be continue configure.

Reply:
Ah, good point.